tests/test_incidents.py

```
@@ -14,7 +13,7 @@
     get_incident_by_id,
     get_last_incidents,
     merge_incidents_to_id,
-    remove_alerts_to_incident_by_incident_id,
+    remove_alerts_to_incident_by_incident_id, enrich_incidents_with_alerts,
```

Ruff

`keep.api.core.db.enrich_incidents_with_alerts` imported but unused

Remove unused import: `keep.api.core.db.enrich_incidents_with_alerts`

(F401)